iceoryx_posh/test/integrationtests/test_posh_mepoo.cpp

```diff
@@ -416,7 +416,7 @@ TEST_F(Mepoo_IntegrationTest, WrongSampleSize)
     constexpr uint32_t SAMPLE_SIZE = 2048U;
     constexpr uint32_t REPETITION = 1U;
     iox::cxx::optional<iox::Error> receivedError;
-    auto errorHandlerGuard = iox::ErrorHandler::setTemporaryErrorHandler(
+    auto errorHandlerGuard = iox::ErrorHandler<iox::Error>::setTemporaryErrorHandler(
```

Comment:
We may be able to get rid of the explicit template argument by defining a specific error handler alias (via using) in the ''posh' namespace. This could actually be good practice and leads to modular error handling without. We would then call the `iox::posh::ErrorHandler` explicitly or implicitly (the latter may be undesirable if the error handler is not configured correctly for all namespaces where it is called).